specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2021-05-13-preview/examples/ScalingPlan_Create.json

```diff
@@ -0,0 +1,163 @@
+{
+  "parameters": {
+    "subscriptionId": "daefabc0-95b4-48b3-b645-8a753a63c4fa",
+    "resourceGroupName": "resourceGroup1",
+    "scalingPlanName": "scalingPlan1",
+    "api-version": "2021-05-13-preview",
+    "scalingPlan": {
+      "location": "centralus",
+      "tags": {
+        "tag1": "value1",
+        "tag2": "value2"
+      },
+      "properties": {
+        "description": "des1",
+        "friendlyName": "friendly",
+        "timeZone": "",
+        "hostPoolType": "Personal",
+        "exclusionTag": "value",
+        "schedules": [
+          {
+            "name": "schedule1",
+            "daysOfWeek": [
+              "Monday",
+              "Tuesday",
+              "Wednesday",
+              "Thursday",
+              "Friday"
+            ],
+            "rampUpStartTime": "11:11",
+            "rampUpLoadBalancingAlgorithm": "DepthFirst",
+            "rampUpMinimumHostsPct": 20,
+            "rampUpCapacityThresholdPct": 80,
+            "peakStartTime": "11:11",
+            "peakLoadBalancingAlgorithm": "BreadthFirst",
+            "rampDownStartTime": "11:11",
+            "rampDownLoadBalancingAlgorithm": "DepthFirst",
+            "rampDownMinimumHostsPct": 20,
+            "rampDownCapacityThresholdPct": 50,
+            "rampDownForceLogoffUsers": true,
+            "rampDownWaitTimeMinutes": 30,
+            "rampDownNotificationMessage": "message",
+            "offPeakStartTime": "11:11",
+            "offPeakLoadBalancingAlgorithm": "DepthFirst"
```

Comment:
@datr93 Thanks for changing the time format. I think it would make sense to have time values the customers might use, for example:
```
            "rampUpStartTime": "06:00",
            "rampUpLoadBalancingAlgorithm": "DepthFirst",
            "rampUpMinimumHostsPct": 20,
            "rampUpCapacityThresholdPct": 80,
            "peakStartTime": "08:00",
            "peakLoadBalancingAlgorithm": "BreadthFirst",
            "rampDownStartTime": "18:00",
            "rampDownLoadBalancingAlgorithm": "DepthFirst",
            "rampDownMinimumHostsPct": 20,
            "rampDownCapacityThresholdPct": 50,
            "rampDownForceLogoffUsers": true,
            "rampDownWaitTimeMinutes": 30,
            "rampDownNotificationMessage": "message",
            "offPeakStartTime": "20:00",
            "offPeakLoadBalancingAlgorithm": "DepthFirst"
```